### PR TITLE
#166879989 Install and configure the react router

### PR DESCRIPTION
**Pull request by Manorlds-Eaglespark**
#### What does this PR do?
Setup react router version 4 (React-router-dom)

#### Description of Task to be completed?
We are going to use react router for our client. Install and configure the router creating a dummy route for login and the "/" page to test the configuration.
Create a components and views file structure where view pages are in the views folder but individual components are in the components folder.

#### How should this be manually tested?
Clone this repo, cd into it and switch to this branch and run the following commands

1. npm install      - to install project dependencies
2. npm start        - to launch the application

#### What are the relevant pivotal tracker stories?
[#166879989](https://www.pivotaltracker.com/story/show/166879989)

Screenshot 2019-07-09 at 19 04 40

### CODE REVIEW

**Review by reifred — CHANGES_REQUESTED** [review]
@Manorlds-Eaglespark thanks for configuring react router. However, kindly use this guide for a better commit message. 
https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices
image

**Comment by Manorlds-Eaglespark** [comment]
> @Manorlds-Eaglespark thanks for configuring react router. However, kindly use this guide for a better commit message.
> https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices
> image

Screenshot 2019-07-09 at 09 07 40
Resolved. Thanks @reifred

**Review by Rhytah — APPROVED** [review]

**Review by Rhytah — APPROVED** [review]

**Review by reifred — APPROVED** [review]

**Review by Emmanuel-Dominic — APPROVED** [review]

File: src/Routes.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```
Comment (Yiga-fred) [inline src/Routes.js]: There is no need for this component to be a class based component @Manorlds-Eaglespark

**Review by Yiga-fred — CHANGES_REQUESTED** [review]

File: src/Routes.js (hunk 1 above)
Comment (Manorlds-Eaglespark) [reply src/Routes.js]: > There is no need for this component to be a class-based component @Manorlds-Eaglespark

Thanks, @Yiga-fred. The implementation has been changed from class-based to function based.

**Review by Yiga-fred — APPROVED** [review]

**Pull request merged**